client/src/components/index.tsx

```diff
@@ -1,6 +1,6 @@
-import Search from "./Search/Search";
+import Search from "./search/Search";
 import Button from "./Button/Button";
-import Input from "./Input/Input";
+import Input from "./input/Input";
```

Comment:
Your local system keeps changing these, both the file and the directory should be capitalized.